modules/tensor_mechanics/doc/theory/capped_weak_plane.tex

```diff
@@ -396,6 +396,7 @@ \chapter{Smoothing}
 
 
 \chapter{Constraints and assumptions concerning parameters}
```

Comment:
Yes, i really want to move all my documentation (there is a lot of PorousFlow stuff too) to the new doco system.  Last time i looked there wasn't quite the functionality that i wanted (maybe it was /label{}  /ref{} for equations?).   That's one thing i want to explore when i'm at INL in a month's time.